test/fixtures/transaction-response.json

```diff
@@ -0,0 +1,89 @@
+{
+  "type": "payment",
+  "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+  "specification": {
+    "source": {
+      "address": "r9cZA1mLK5R5Am25ArfXFmqgNwjZgnfk59",
+      "amount": {
+        "currency": "XRP",
+        "value": "1.112209"
+      }
+    },
+    "destination": {
+      "address": "rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM",
+      "amount": {
+        "currency": "USD",
+        "value": "0.001",
+        "counterparty": "rMH4UxPrbuMa1spCBR98hLLyNJp4d8p4tM"
+      }
+    },
+    "fee": "0.00001",
```

Comment:
Oh, I see. Yeah, we want to be uniform, but also concise and non-redundant. Since the fee is always XRP, it would be redundant to always label it as XRP.